CraftTweaker2-API/src/main/java/crafttweaker/api/recipes/IRecipeManager.java

```diff
@@ -55,6 +55,19 @@
     @ZenMethod
     void addShaped(String name, IItemStack output, IIngredient[][] ingredients, @Optional IRecipeFunction function, @Optional IRecipeAction action);
     
+    /**
+     * Adds a shaped recipe specifying group.
+     *
+     * @param group       recipe group
+     * @param name        recipe name
+     * @param output      recipe output
+     * @param ingredients recipe ingredients
+     * @param function    recipe function (optional)
```

Comment:
Restating the parameter names is counterproductive.

Reply:
I'm not 100% up-to-date on how the documentation system works but I presumed that each function needed to specify its parameters. This is just the documentation for the current addShaped function with the group specified.